### PR TITLE
correct the local policy e2e test

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: test/pkg/e2e/hoh-local-policy_test.go [hunk 1]
```diff
@@ -31,136 +32,181 @@ const (
...
-					utils.NewPolicyGVR()).Namespace(LOCAL_POLICY_NAMESPACE).Get(
-					context.TODO(), LOCAL_POLICY_NAME, metav1.GetOptions{})
-				Expect(err).ShouldNot(HaveOccurred())
-				unstructedContent := unstrcutedPolicy.UnstructuredContent()
-				var policy policiesv1.Policy
-				err = runtime.DefaultUnstructuredConverter.FromUnstructured(unstructedContent, &policy)
-				Expect(err).ShouldNot(HaveOccurred())
-				if len(policy.Status.Status) <= 0 {
-					return fmt.Errorf("inform local policy status is not ready")
+				policyStr, err := clients.Kubectl(clients.HubClusterName(), "exec", "-i", postgresMaster, "-c", "database",
```
Comment (morvencao) [inline test/pkg/e2e/hoh-local-policy_test.go:166]: why not use postgresql client to access DB?

Comment (yanmxa) [reply test/pkg/e2e/hoh-local-policy_test.go:166]: Currently, the host of `postgreURI` is made of `clusterIP`, which means we can't access it from outside the cluster. If we use the postgresql client, we have to create another `NodePort` service to expose it.

Comment (yanmxa) [reply test/pkg/e2e/hoh-local-policy_test.go]: @morvencao I have started a `nodeport` of the postgres cluster to enable the PostgreSQL client.

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**